### PR TITLE
[Club21-41] Fix UI of Club21

### PR DESCRIPTION
**Pull request by ttduongtran**
Restyle UI

### CODE REVIEW

File: components/DashboardComponent.js [hunk 1]
```diff
@@ -124,102 +180,110 @@ const DashboarComponent = ({ t }) => {
...
     },
     breakpoints: {
-      1280: {
-        slidesPerView: 4,
-        spaceBetween: 40
+      200: {
+        slidesPerView: 1,
+        spaceBetween: 10
+      },
+      575: {
```
Comment (thaind97git) [inline components/DashboardComponent.js]: image

File: layouts/SimpleTableLayout.js [hunk 2]
```diff
@@ -38,7 +47,7 @@ const SimpleTableLayout = ({
 }) => {
   const classes = useStyles();
   return (
-    <div className={classes.table}>
+    <div className={clsx(classes.table, classes.tableResposive)}>
```
Comment (thaind97git) [inline layouts/SimpleTableLayout.js]: You can remove this, because in [this PR](https://github.com/Rev-Tech/club21js/pull/8/files) already to fix them.

Comment (ttduongtran) [reply layouts/SimpleTableLayout.js]: Okie Thai

File: components/DashboardComponent.js (hunk 1 above)
Comment (ttduongtran) [reply components/DashboardComponent.js]: image

It was fixed. 
However some cases, it will be down line if it has long text

**Comment by thaind97git** [comment]
image

I see the `hr tag bottom of header` don't have width: 100%. Can you fix them?

**Comment by ttduongtran** [comment]
> image
> 
> I see the `hr tag bottom of header` don't have width: 100%. Can you fix them?

Thanks. I fixed it

**Pull request merged**